package.json

```diff
@@ -6,10 +6,12 @@
     "@babel/polyfill": "^7.6.0",
     "@material-ui/core": "^4.5.1",
     "@material-ui/icons": "^4.5.1",
+    "@types/lodash": "^4.14.156",
     "@types/react-html-parser": "^2.0.1",
     "@types/react-swipeable-views": "^0.13.0",
     "classnames": "^2.2.6",
     "localized-strings": "^0.2.4",
+    "lodash": "^4.17.15",
```

Comment:
Was lodash needed? If not, remove

Reply:
It wasn't needed so I removed it. Hopefully this doesn't affect lodash that was installed by some of the other libraries as their dependency.